event/event.gen.go

```diff
@@ -376,7 +382,7 @@ func DurationChange(listener func(*vecty.Event)) *vecty.EventListener {
 
 // Emptied is an event fired when the media has become empty; for example, this
 // event is sent if the media has already been loaded (or partially loaded),
-// and the load() method is called to reload it.
+// and the load() method is called to reload it.
```

Comment:
We should figure out what changed here. This may be the MDN using another weird space unicode character, and if so, we should swap it out in the generator with a plain ASCII space character.

Reply:
Probably copying https://github.com/gernest/vecty/blob/5f529e0c4616b201de3fc21c4c8873c92862bf89/elem/generate.go#L158-L160 into this generator would do the trick.

Reply:
> We should figure out what changed here.

I already checked. MDN changed a non-breaking whitespace (U+00A0) for a normal whitespace (U+0020).

![image](https://user-images.githubusercontent.com/1924134/38785683-65eef946-40f0-11e8-896a-19059310a56b.png)

Copying `desc = strings.Replace(desc, "\u00a0", " ", -1)` might still be desirable if you really want to eliminate those non-breaking whitespaces, in case they come up in the future or elsewhere.

However, I think having non-breaking whitespace in documentation isn't a big deal. I don't see a reason to actively avoid them.

Reply:
Got it. This change LGTM as-is then.

Unrelated to this change, but I do want to avoid non-breaking whitespace in documentation because it easily leads to changes like this where there is uncertainty about why a line has changed.